tutorials/tracing/manual_instrumentation_tutorial.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    # becuase we are using Open AI, we will use this along with our custom instrumentation
</pre><p>typo in "because"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>